engine/pkg/externalresource/model/model.go

```diff
@@ -157,3 +157,8 @@ func ParseResourcePath(rpath ResourceID) (ResourceType, ResourceName, error) {
 	suffix := path.Join(segments[1:]...)
 	return resourceType, suffix, nil
 }
+
+// ParseResourceID returns an ResourceID based on given ResourceType and ResourceName.
+func ParseResourceID(rtype ResourceType, name ResourceName) ResourceID {
```

Comment:
The `Parse` word in this function name is not accurate. The function `ParseResourcePath` is more likely `PasreResourceID`.

Reply:
How about `GenXXX` or `BuildXXX`?